test/integration/websocket_integration_test.cc

```diff
@@ -129,6 +129,7 @@ void WebsocketIntegrationTest::initialize() {
               *bootstrap.mutable_static_resources()->mutable_clusters(0), protocol_options);
         });
   } else if (upstreamProtocol() == Http::CodecType::HTTP3) {
+    Runtime::maybeSetRuntimeGuard("envoy.reloadable_features.use_http3_header_normalisation", true);
```

Comment:
I'm slightly surprised we don't need to do this anywhere else?

Reply:
I think it's because upgrade is a very niche feature?